pkgs/applications/networking/misc/zammad/default.nix

```
@@ -130,7 +131,16 @@ stdenv.mkDerivation {
     node_modules=${yarnEnv}/libexec/Zammad/node_modules
     ${yarn2nix-moretea.linkNodeModulesHook}
 
+    mkdir redis-work
+    pushd redis-work
+    ${redis}/bin/redis-server &
```

```suggestion
    redis-server &
```
lets make this cross friendly by adding redis to nativeBuildInputs

---

Done.